demos/projects/NXP/mimxrt1060/CMakeLists.txt

```diff
@@ -110,3 +111,24 @@ add_custom_command(TARGET ${PROJECT_NAME}
     COMMAND ${CMAKE_OBJCOPY} -Obinary $<TARGET_FILE:${PROJECT_NAME}> ${PROJECT_NAME}.bin
     COMMENT "Generate Bin file"
     VERBATIM)
+
+target_link_libraries(${PROJECT_NAME}-pnp PRIVATE
```

Comment:
add check for this binary in ci_test.sh

https://github.com/Azure-Samples/iot-middleware-freertos-samples/blob/328fd10cd0648c25494bb35cdff1f5a3e1e03cbf/.github/scripts/ci_tests.sh#L38

for all ports